src/components/shared/ReviewsCard.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import tyson from '../../assets/images/tyson.svg';
+import Stars from './Stars';
+
+type ReviewsProps = {
+  image?: string;
+  name?: string;
+  rating: number;
```

Comment:
Hello @SergeInGithub, I don't think it is good to display rating by Stars and again display number of rating

Reply:
The inspiration for this was got from the figma design, and we'll also later on use the rating to display the average number ratings @emilenfc